packages/components/src/alignment-matrix-control/test/index.tsx

```diff
@@ -17,34 +17,130 @@ const getCell = ( name: string ) => {
 	return within( getControl() ).getByRole( 'gridcell', { name } );
 };
 
+const asyncRender = async ( jsx: any, focusedCell = 'center center' ) => {
```

Comment:
I think the name of this function makes less and less sense - we don't really async render; we render synchronously, but we wait for the composite state to initialize. So could we rename this function to reflect that behavior? Perhaps something along the lines of: `renderAndInitCompositeStore`.

Reply:
Yeah makes sense